docs/current/README.md

```diff
@@ -53,49 +53,48 @@ Vaquero can run in multiple modes: `server`, `agent`, and `standalone`. This con
 **sample-standalone-config.yaml**
 ************************************************************
 ```
----
 ServerApi:
- Address: "127.0.0.1"
- Port: 24601
+  Address: "127.0.0.1"
+  Port: 24601
 AgentApi:
- InsecureAddr: "127.0.0.1"
- InsecurePort: 24602
+  InsecureAddr: "127.0.0.1"
+  InsecurePort: 24602
 AssetServer:
- Addr: "127.0.0.1"
- Port: 8080
- BaseDir: "/tmp/vaquero/files"
- Scheme: http
-DHCPMode: server
-DHCPCIDR: "127.0.0.1/16"
+  Addr: "127.0.0.1"
+  Port: 8080
+  BaseDir: "/tmp/vaquero/files"
+  Scheme: http
+DHCP:
```

Comment:
Not exactly. Change this to JUST be 
- DHCPMode: 
(i.e. revert to the old version, but remove DHCPCIDR). See https://github.com/CiscoCloud/vaquero/issues/333 for further clarification.